core/src/main/java/lucee/runtime/functions/system/InternalRequest.java

```diff
@@ -78,9 +87,9 @@ public static Struct call(final PageContext pc, String template, String method,
 		Charset reqCharset = StringUtil.isEmpty(strCharset) ? pc.getWebCharset() : CharsetUtil.toCharset(strCharset);
 
 		String ext = ResourceUtil.getExtension(template, null);
-		// welcome files
+		// template
 		if (StringUtil.isEmpty(ext)) {
-			throw new FunctionException(pc, "Invoke", 1, "url", "welcome file listing not supported, please define the template name.");
+			throw new FunctionException(pc, "_InternalRequest", 1, "template", "template path is invalid");
 		}
```

Comment:
For Lucee 6 we did add "InternalRequest" asn an alias for "_InternalRequest" so when we do the same for 6 we need to change the name there

Reply:
I've changed name to InternalRequest https://github.com/lucee/Lucee/commit/b30c0de34351b6babfbc0ad4e3e014c88a33205f